src/pyBiodatafuse/annotators/bgee.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import datetime
+import os
+from string import Template
+
+import pandas as pd
+from SPARQLWrapper import JSON, SPARQLWrapper
+
+from pyBiodatafuse.utils import collapse_data_sources, get_identifier_of_interest
+
+
+def get_version_bgee() -> dict:
+    # not sure if a version per-se can be retrieved, but the endpoint supports
+    # http://purl.org/dc/terms/modified
```

Comment:
Is this the last time the whole database has been updated?

Reply:
yes, basically it's equivalent to the "version" since there are only a couple of releases per year